projects/fal/src/fal/apps.py

```diff
@@ -15,12 +15,29 @@
 _REALTIME_URL_FORMAT = f"wss://{flags.FAL_RUN_HOST}/{{app_id}}"
 
 
-def _backwards_compatible_app_id(app_id: str) -> str:
-    if "/" not in app_id:
-        # Convert the app_id to the format used in the URL.
-        return app_id.replace("-", "/", 1)
-
-    return app_id
+def _backwards_compatible_app_id(app_id: str, path: str = "/") -> tuple[str, str]:
```

Comment:
can you check how the fal-js client do it? i think it'd be nice if we can keep them in sync

Reply:
copied the solution that is done in js client